packages/contracts/test/contracts/L1/messaging/L1StandardBridge.spec.ts

```diff
@@ -564,4 +575,10 @@ describe('L1StandardBridge', () => {
       )
     })
   })
+
+  describe('donateETH', () => {
+    it('it should just call the function', async () => {
+      await expect(L1StandardBridge.donateETH()).to.not.be.reverted
```

Comment:
Could also confirm here that when ETH value is sent alongside the tx, it doesn't revert.